src/Payment/OrderLines.php

```diff
@@ -188,7 +188,7 @@ private function process_fees()
     {
         if (! empty($this->order->get_items('fee'))) {
             foreach ($this->order->get_items('fee') as $cart_fee) {
-                if ($cart_fee['tax_status'] === 'taxable' && $cart_fee['total_tax'] > 0) {
```

Comment:
Any ideas why this `> 0` check was added before?

Reply:
I guess the previous developer did not consider negative fees as possible in the WooCommerce workflow. Now we know that some plugins use negative fees to apply discounts to the order.